publications-2018.bib

```diff
@@ -922,4 +922,12 @@ @ARTICLE{Freddi2018573
   pages={573-592}
 }
 
+@mastersthesis{schneider2018,
+  author    = {Schneider, David},
+  title     = {Simulation von Fluid-Struktur-Interaktion mit der Kopplungsbibliothek preCICE},
```

Comment:
I am sorry that I keep finding stuff, but we should at least have `{preCICE}` here to get the capitalization correct, right? I think we are more lax on other capitalization issues.

Reply:
I mulled this at the other entry I changed, but then realized that it probably doesn't matter. The issue is that bibtex generally converts everything into lower-case for the titles of articles, and consequently if one wants something capitalized, it needs to be enclosed in `{...}`. 

But bibtex doesn't do that for book titles, which are copied verbatim to latex. I suspect that that's also the case for thesis titles, and that consequently the changes I made in the other entry are pointless (though also harmless).

Let's go with this version and I'll look at how it comes out. If it's wrong, I'll fix it up in a follow-up.

Reply:
Sounds good.

Reply:
It looks correct on the website!